uber/models.py

```diff
@@ -2086,7 +2089,7 @@ def register_session_listeners():
     """
     listen(Session.session_factory, 'before_flush', _acquire_badge_lock)
     listen(Session.session_factory, 'before_flush', _presave_adjustments)
-    listen(Session.session_factory, 'before_flush', _track_changes)
+    listen(Session.session_factory, 'after_flush', _track_changes)
```

Comment:
Well, our docstring specifically says `NOTE 2: The order in which we register these listeners matters.` so I sure hope it's right, hahaha. o.o